src/style.css

```diff
@@ -1 +1,15 @@
-/* styles go here */
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.label {
+  display: block;
+}
+
+.label:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+#cars {
+  overflow: hidden;
+}
```

Comment:
Ok, but usually we use classes for styling
You may leave it as it is now, just point for future